internal/injector/builtin/yaml/directive/orchestrion-enabled.yml

```diff
@@ -17,8 +17,10 @@ aspects:
     tracer-internal: true # This is safe to apply in the tracer itself.
     join-point:
       all-of:
-        - directive: dd:orchestrion-enabled
         - value-declaration: bool
+        - one-of:
+            - declaration-of: github.com/DataDog/orchestrion/built.WithOrchestrion
+            - directive: dd:orchestrion-enabled
```

Comment:
Can you change the name of the aspect ? because it's not self-encompassing anymore